tests/test.py

```diff
@@ -512,6 +514,124 @@ def testZstashWithHPSS(self):
 
         cleanup(hpss_path)
 
+    def testZstashWithoutHPSS(self):
```

Comment:
Is this correctly testing everything we care about and nothing we don't care about?

Reply:
- Can you add a test for 'zstash extract' in testZstashWithoutHPSS?
- In 'testKeepTars' I would add the following steps
  - delete files (but not zstash files)
  - extract everything, make sure there are not errors
  - make sure tar files are still there
  - check that tar files are read-only

Reply:
@golaz What is the expected behavior of `zstash extract` if there is no HPSS available to extract from? When I try running `zstash extract` I get `sqlite3.OperationalError: unable to open database file`.

Reply:
zstash extract (with the zstash/ subdirectory present and containing index.db and tar files) should extract all the files from the local tar files.